goreq.go

```diff
@@ -370,6 +380,10 @@ func (r Request) addHeaders(headersMap http.Header) {
 	headersMap.Add("Content-Type", r.ContentType)
 }
 
+func (r Request) Debug(is_debug bool) {
```

Comment:
I don't fully understand this function, you're not using it anywhere and you're setting the r.ShowDebug flag to a `copy` of the Request value. 

If what you're trying to accomplish is to set the ShowDebug property to the Request instance, then you associate the Debug method to a pointer of Request, otherwise it won't work.